src/AltinnCore/Runtime/RestControllers/ProcessController.cs

```diff
@@ -552,6 +556,12 @@ private List<InstanceEvent> ChangeProcessStateAndGenerateEvents(Instance instanc
                 events.Add(GenerateProcessChangeEvent("process:StartTask", instance, now));
             }
 
+            if (previousElementId != null && previousElementId.Equals("FormFilling_1"))
```

Comment:
Is it possible to determine that generating a pdf is required in a more generic way that won't require changing the code when the process allows for more that one step?

Reply:
This does allow the process to have more than once step. It just triggers a generation if the current step is formFilling_1.

Reply:
Or is it something I'm not seeing?

Reply:
What happens if we introduce a new step FormFilling_2 and would like to have a pdf generated when that step is completed too? Or isn't that a likey case?

Reply:
Oh, gotcha. Yes, then we would have to check if the current step is equal to either of those two. But I agree that we should have a more generic way of adding these kinds of events. The questions is more if this is something we can live with for MVP I guess.